lib/participant.js

```diff
@@ -296,39 +296,64 @@ class Participant extends EventEmitter {
       const publication = new RemoteTrackPublication(signaling, { log });
       self._addTrackPublication(publication);
 
-      signaling.getTrackTransceiver().then(trackReceiver => {
-        const RemoteTrack = {
-          audio: RemoteAudioTrack,
-          video: RemoteVideoTrack,
-          data: RemoteDataTrack
-        }[signaling.kind];
-
-        // NOTE(mroberts): It should never be the case that the TrackSignaling and
-        // MediaStreamTrack or DataTrackReceiver kinds disagree; however, just in
-        // case, we handle it here.
-        if (!RemoteTrack || signaling.kind !== trackReceiver.kind) {
-          return;
-        }
-
-        const track = signaling.kind === 'data'
-          ? new RemoteTrack(trackReceiver, { log, name: signaling.name })
-          : new RemoteTrack(trackReceiver, signaling.isEnabled, { log, name: signaling.name });
+      let isSubscribed = signaling.isSubscribed;
```

Comment:
Nitpick: I think we can prefer `const` here. Also, we could destructure to avoid repeating "isSubscribed".

Reply:
@markandrus `isSubscribed` is updated every time "updated" is fired. So we should use `let` here.

Reply:
Ah, right 👍